docker/development/Dockerfile

```
@@ -106,12 +104,11 @@ RUN apt-get update -y \
    python3-pip \
    && rm -rf /var/lib/apt/lists/*
 
-RUN python3.8 -m pip install --upgrade pip ipython \
-   && python3.9 -m pip install --upgrade pip ipython \
+RUN && python3.9 -m pip install --upgrade pip ipython \
```

Hadolint (2.12.0)

[error] Unexpected start of line. If breaking lines, |/||/&& should be at the end of the previous one.

(SC1133)